conductor/src/main/java/com/bluelinelabs/conductor/Controller.java

```diff
@@ -1082,7 +1082,7 @@ private void removeViewReference(@Nullable Context context) {
     final View inflate(@NonNull ViewGroup parent) {
         if (view != null && view.getParent() != null && view.getParent() != parent) {
             detach(view, true, false);
-            removeViewReference(view.getContext());
+            removeViewReference(view != null ? view.getContext() : null);
```

Comment:
Wouldn't a more correct behavior to capture the view locally in the beginning of the method body?

Reply:
This is a good call. Doesn't matter in this case because the context is only used when the controller is being destroyed and it will never be getting destroyed during `inflate`. But for the sake of correctness and avoiding future footguns I'll update it to work this way.